libs/dal/src/repositories/integration/types.ts

```diff
@@ -0,0 +1,7 @@
+export type KindId = string;
+
+export enum ProviderKindEnum {
+  ENVIRONMENT = 'environment',
+  TENANT = 'tenant',
+  WORKFLOW = 'workflow',
+}
```

Comment:
So far we will only use the environment kind but ready for the future.

Reply:
I'm probably jumping ahead - but maybe already add this type to shared ? As we would use it everywhere for sure

Reply:
Good point. But going to delete it as we do not need it until we create the `conditions` in the future.